integration_tests/pages/booking.ts

```diff
@@ -0,0 +1,15 @@
+import Page, { PageElement } from './page'
+
+export default class BookingsPage extends Page {
+  constructor() {
+    super('Booking')
+  }
+
+  visitDate = (): PageElement => cy.get('[data-test=visit-date]')
```

Comment:
```suggestion
  visitDate = (index: number): PageElement => cy.get(`[data-test=visit-date-${index.toString()}]`)
```
Think you might need this to fix integration tests.